soc/xtensa/intel_adsp/cavs/include/intel_tgl_adsp/adsp_memory.h

```diff
@@ -25,6 +25,8 @@
 /* Host shared memory windows */
 #define HP_SRAM_WIN0_BASE (L2_SRAM_BASE + CONFIG_ADSP_WIN0_OFFSET)
 #define HP_SRAM_WIN0_SIZE 0x2000
+#define HP_SRAM_WIN2_BASE (HP_SRAM_WIN0_BASE + 0x4000)
+#define HP_SRAM_WIN2_SIZE 0x2000
```

Comment:
FWIW: I tried at one point to switch this API to a less hard-coded one so you could just have e.g. "`char window2[WIN2_SIZE]` somewhere and have the standard linker locate and pack them.  There's a lot of wasted space (and no small amount of linker complexity) right now to reserve all this memory.  But there were enough spots in SOF still using the constants that I was too scared.  Might be worth considering as this code evolves though.

Reply:
As noted in my comment in 49077 (gdb stub), we need to keep the configurability, but the linker cleanup might be doable. We are now in the process of moving to native Zephyr drivers for the SOF app, and once that is complete, there shouldn't be any users of these offsets left on SOF side.